pyproject.toml

```diff
@@ -10,6 +10,7 @@ keywords = ["differential privacy", "data analysis", "apache beam", "pyspark", "
 dependencies = [
     "python-dp (>=1.1.5rc4,<2.0.0)",
     "numpy (>=1.20.1,<2.0.0)",
+    "dill (>=0.3.7)",
```

Comment:
do we need dill? (I think explicitly PipelineDP doesn't use it, but Beam or Spark might use it, but then it's should be installed automatically as transitive dependency)